### PR TITLE
Initial implementation of Command transaction family

### PR DESCRIPTION
**Pull request by shannynalayna**
Implements the Command transaction family through the workload module. Adds Commands defined by the Command Family RFC that were not previously implemented. Also adds tests for these commands via the static execution adapter.

### CODE REVIEW

File: libtransact/src/workload/command.rs [hunk 1]
```diff
@@ -159,13 +197,13 @@ impl std::str::FromStr for Command {
         let command = command_parts.next().map(|s| s.to_lowercase());
         match command {
             Some(ref set) if set == "set" => {
-                let address = command_parts
```
Comment (peterschwarz) [inline libtransact/src/workload/command.rs]: All of this parsing should now be replaced with just a `protobuf::parse_from_bytes`

File: libtransact/src/workload/command.rs [hunk 2]
```diff
@@ -75,24 +96,41 @@ impl TransactionHandler for CommandTransactionHandler {
...
-    Get { address: String },
-    Fail { error_msg: String },
-}
-
 impl std::fmt::Display for Command {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
         match self {
-            Command::Set {
-                ref address,
+            Command::SetState {
```
Comment (peterschwarz) [inline libtransact/src/workload/command.rs]: This display implementation is just for rendering these commands for the tests also replaced by the protobuf classes.

File: libtransact/protos/command.proto [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+message AddReceiptData {
+    // a non-empty byte array
+    bytes receipt_data = 1;
+}
+
+// The duration and method of sleep to apply during a SLEEP command.
+message Sleep {
+    enum SleepType {
+        // This will wait, releasing cpu utilization to other processes
+        WAIT = 0;
```
Comment (ltseeley) [inline libtransact/protos/command.proto]: Maybe "IDLE" and "BUSY" would be better?

File: libtransact/src/protocol/command.rs [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------------
+ */
+
+/// A command to be executed
+#[derive(Debug)]
+pub enum Command {
+    SetState {
+        state_keys: String,
```
Comment (ltseeley) [inline libtransact/src/protocol/command.rs]: Should this be `state_key`? Same below.

File: libtransact/src/protocol/command.rs [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    },
+    DeleteState {
+        state_keys: String,
+    },
+    GetState {
+        state_keys: String,
+    },
+    AddEvent {
+        event_type: String,
+        attributes: Vec<(String, String)>,
```
Comment (ltseeley) [inline libtransact/src/protocol/command.rs]: Should this be `Vec<(String, Vec<u8>)>`?

File: libtransact/protos/command.proto (hunk 3 above)
Comment (shannynalayna) [reply libtransact/protos/command.proto]: This is the SleepType that was defined in the Command family RFC.

**Pull request merged**